src/test/ui/lifetime-errors/ex3-both-anon-regions-both-are-structs-3.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0623]: lifetime mismatch
+  --> $DIR/ex3-both-anon-regions-both-are-structs-3.rs:16:11
+   |
+15 | fn foo(mut x: Ref) {
+   |               ---
+   |               |
+   |               these two types are declared with different lifetimes...
```

Comment:
eng: could you change this wording to "this type was declared with multiple lifetimes"?